content/sensu-go/6.0/reference/entities.md

```diff
@@ -66,6 +66,36 @@ You can modify the proxy entity later if needed.
 
 Use [proxy entity filters][19] to establish a many-to-many relationship between agent entities and proxy entities if you want even more power over the grouping.
 
+## Create and modify agent entities
+
+When an agent connects to a backend, the agent entity definition is created from the information in the `agent.yml` configuration file.
+The default `agent.yml` file location depends on your operating system:
+
+- Linux and macOS: `/etc/sensu/agent`
+- Windows: `C:\ProgramData\sensu\config`
```

Comment:
Maybe we can point them to the `config-file` attribute in https://docs.sensu.io/sensu-go/latest/reference/agent/#general-configuration-flags. That has all the default file paths documents already!